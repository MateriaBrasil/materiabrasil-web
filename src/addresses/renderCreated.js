import React from 'react';
import Redirect from 'react-router-dom/Redirect';

export default props => () => {
  const { slug } = props.currentUser.suppliers[0];
  return <Redirect to={`/suppliers/${slug}?reload=true`} />;
};
