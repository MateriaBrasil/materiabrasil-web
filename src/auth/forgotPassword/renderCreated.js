import React from 'react'

import Redirect from 'react-router-dom/Redirect'

export default props => created => {
  return <Redirect to="/auth/forgot-password/sent" />
}
