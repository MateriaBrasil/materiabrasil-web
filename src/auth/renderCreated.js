import React from 'react'

import Created from './Created'

export default props => user => <Created {...props} created={user} />
