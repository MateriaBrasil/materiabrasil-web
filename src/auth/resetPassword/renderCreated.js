import { navigate } from '@reach/router'

export default props => created => {
  navigate(`/auth/sign-in`)
}
