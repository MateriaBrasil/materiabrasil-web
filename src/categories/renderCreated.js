import React from 'react'

import Created from './Created'

export default props => created => <Created {...props} created={created} />
