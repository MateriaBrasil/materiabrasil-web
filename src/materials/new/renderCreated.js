import React from 'react'
import Redirect from 'react-router-dom/Redirect'

export default props => material => <Redirect to={`/${material.id}`} />
