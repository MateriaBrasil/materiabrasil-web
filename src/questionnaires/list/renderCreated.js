import React from 'react'
import List from './List'

export default props => ({ create }) => {
  return <List {...props} />
}
