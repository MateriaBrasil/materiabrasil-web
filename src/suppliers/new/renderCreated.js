import React from 'react'
import Created from './Created'

export default props => supplier => <Created {...props} supplier={supplier} />
