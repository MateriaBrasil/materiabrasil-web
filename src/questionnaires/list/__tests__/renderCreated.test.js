import React from 'react'
import renderer from 'react-test-renderer'

import renderCreated from '../renderCreated'

jest.mock('../Created', () => props => <div {...props}>Created</div>)

it('render correctly', () => {
  const tree = renderer
    .create(renderCreated({ foo: 'bar' })({ questionId: 1234 }))
    .toJSON()
  expect(tree).toMatchSnapshot()
})
