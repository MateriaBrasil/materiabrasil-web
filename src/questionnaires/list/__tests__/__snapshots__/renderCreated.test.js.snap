// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly 1`] = `
<div
  created={
    Object {
      "questionId": 1234,
    }
  }
  foo="bar"
>
  Created
</div>
`;
